sql/migrations/state/0007_next.sql

```diff
@@ -1,3 +1,18 @@
 ALTER TABLE activesets ADD epoch INT DEFAULT 0 NOT NULL;
 CREATE INDEX activesets_by_epoch ON activesets (epoch asc);
 UPDATE activesets SET epoch = 7 WHERE epoch = 0;
+DROP INDEX rewards_by_layer;
```

Comment:
i think 0007 was released. we should tag it as released (or atleast in the release candidate branch) and write new migration code to incremented version (0008)

Reply:
Yes, this is my bad. I will update the `RELEASE.md` on in a separate PR to include information that this needs to be done when releasing a new version.

Reply:
PR: https://github.com/spacemeshos/go-spacemesh/pull/5250/